app/Http/Kernel.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Pterodactyl\Http;
 
+use Pterodactyl\Http\Middleware\MaintenanceMiddleware;
```

Comment:
Run the linter against these files, these imports should be in length based order (and then alphabetical if needed).

Reply:
I told him to leave it as style ci will fix it automatically when we run it again, no real need to do an extra commit for such little stuff imo

Reply:
Ah didn't notice the styleci failure.